source/lac/lapack_full_matrix.cc

```diff
@@ -88,13 +88,39 @@ LAPACKFullMatrix<number>::grow_or_shrink (const size_type n)
   TransposeTable<number> copy(*this);
   const size_type s = std::min(std::min(this->m(), n), this->n());
   this->TransposeTable<number>::reinit (n, n);
-  for (unsigned int i = 0; i < s; ++i)
-    for (unsigned int j = 0; j < s; ++j)
+  for (size_type i = 0; i < s; ++i)
+    for (size_type j = 0; j < s; ++j)
       (*this)(i,j) = copy(i,j);
 }
 
 
 
+template <typename number>
+void
+LAPACKFullMatrix<number>::remove (const size_type row, const size_type col)
+{
+  Assert (row<this->n_rows(), ExcIndexRange(row,0,this->n_rows()));
+  Assert (col<this->n_cols(), ExcIndexRange(col,0,this->n_cols()));
+
+  TransposeTable<number> copy(*this);
```

Comment:
Do we have move constructors? If so, maybe use
```
TransposeTable<number> copy(std::move(*this));
```
The memory is going to be released next anyway, so we might as well steal it.